File: test-explorer/v2x_net452_DotNetSdk/BasicTests.cs
using Xunit;

public class BasicTests
{
    [Fact]
    public void Passing()
    {
        Assert.True(true);
    }

    [Fact]
    public void Failing()
    {
        Assert.True(false);
    }

    [Theory]
    [InlineData(2112)]
    [InlineData("hello from x2x_net452_DotNetSdk")]
    [InlineData(null)]
    public void Theory(object value)
    {
        Assert.NotNull(value);
    }
}

File: test-explorer/v2x_net472_FSharp/BasicTests.fs
namespace FSharp

open Xunit

type BasicTests() =
    [<Fact>]
    member _.Passing() =
        Assert.True true

    [<Fact>]
    member _.Failing() =
        Assert.True false

    [<Theory>]
    [<InlineData("hello from v2x_net472_FSharp")>]
    [<InlineData("")>]
    [<InlineData(null)>]
    member _.Theory(value:string) =
        Assert.NotNull value
